packages/editor/src/components/document-bar/index.js

```diff
@@ -0,0 +1,175 @@
+			const _templateId = getCurrentTemplateId();
+			return {
+				isEditingTemplate:
+					!! _templateId && getRenderingMode() === 'template-only',
+				templateId: _templateId,
+				postType: getCurrentPostType(),
+				postId: getCurrentPostId(),
+			};
+		},
+		[]
+	);
+	const { getEditorSettings } = useSelect( editorStore );
+	const { setRenderingMode } = useDispatch( editorStore );
+
+	return (
+		<BaseDocumentActions
+			postType={ isEditingTemplate ? 'wp_template' : postType }
+			postId={ isEditingTemplate ? templateId : postId }
+			onBack={
+				isEditingTemplate
+					? () =>
+							setRenderingMode(
+								getEditorSettings().defaultRenderingMode
+							)
+					: undefined
+			}
+		/>
+	);
+}
+
+function BaseDocumentActions( { postType, postId, onBack } ) {
+	const { open: openCommandCenter } = useDispatch( commandsStore );
+	const { editedRecord: document, isResolving } = useEntityRecord(
+		'postType',
+		postType,
+		postId
+	);
+	const { templateIcon, templateTitle } = useSelect( ( select ) => {
+		const { __experimentalGetTemplateInfo: getTemplateInfo } =
+			select( editorStore );
+		const templateInfo = getTemplateInfo( document );
+		return {
+			templateIcon: templateInfo.icon,
+			templateTitle: templateInfo.title,
+		};
+	} );
+	const isNotFound = ! document && ! isResolving;
+	const icon = icons[ postType ] ?? pageIcon;
+	const [ isAnimated, setIsAnimated ] = useState( false );
+	const isMounting = useRef( true );
+	const isTemplate = [ 'wp_template', 'wp_template_part' ].includes(
+		postType
+	);
+
+	useEffect( () => {
+		if ( ! isMounting.current ) {
+			setIsAnimated( true );
+		}
+		isMounting.current = false;
+	}, [ postType, postId ] );
+
+	const title = isTemplate ? templateTitle : document.title;
+
+	return (
+		<div
+			className={ classnames( 'editor-document-bar', {
+				'has-back-button': !! onBack,
+				'is-animated': isAnimated,
+			} ) }
+		>
+			{ onBack && (
+				<Button
+					className="editor-document-bar__back"
+					icon={ isRTL() ? chevronRightSmall : chevronLeftSmall }
+					onClick={ ( event ) => {
+						event.stopPropagation();
+						onBack();
+					} }
+					size="compact"
+				>
+					{ __( 'Back' ) }
+				</Button>
+			) }
+			{ isNotFound && <Text>{ __( 'Document not found' ) }</Text> }
+			{ ! isNotFound && (
+				<Button
+					className="editor-document-bar__command"
+					onClick={ () => openCommandCenter() }
+					size="compact"
+				>
+					<HStack
+						className="editor-document-bar__title"
+						spacing={ 1 }
+						justify="center"
+					>
+						<BlockIcon icon={ isTemplate ? templateIcon : icon } />
+						<Text
+							size="body"
+							as="h1"
+							aria-label={
+								typeLabels[ postType ]
+									? // eslint-disable-next-line @wordpress/valid-sprintf
+									  sprintf( typeLabels[ postType ], title )
+									: undefined
+							}
+						>
+							{ title }
+						</Text>
+					</HStack>
+					<span className="edit-site-document-actions__shortcut">
```

Comment:
```suggestion
					<span className="editor-document-bar__shortcut">
```

This change should apply the intended style, change the color from purple to the default color, and prevent the shortcut from appearing in the mobile viewport.